pytorch_lightning/accelerators/accelerator.py

```diff
@@ -362,7 +358,7 @@ def setup_precision_plugin(self) -> None:
         model, optimizers, schedulers = self.precision_plugin.connect(self.model, self.optimizers, self.lr_schedulers)
         self.model = model
         self.optimizers = optimizers
-        self.schedulers = schedulers
+        self.lr_schedulers = schedulers
```

Comment:
We use `self.lr_scheduler` everywhere else. I assume it's a typo but unrelated to this PR